libraries/BLE/examples/central/central.ino

```diff
@@ -0,0 +1,94 @@
+
+#include "ArduinoBLE.h"
+#include "BLEAttribute.h"
+#include "BLECharacteristicImp.h"
+#include "BLEProfileManager.h"
+
+// LED pin
+#define LED_PIN   13
+
+void setup() {
+    Serial.begin(115200);
+    Serial.println("test---");
+    
+    // set LED pin to output mode
+    pinMode(LED_PIN, OUTPUT);
+
+    // begin initialization
+    BLE.begin();
+    Serial.println(BLE.address());
+    
+    BLE.startScanning("LED");
+}
+
+void controlLed(BLEDevice &peripheral)
+{
+    static bool discovered = false;
+    // connect to the peripheral
+    Serial.print("Connecting ... ");
+    Serial.println(peripheral.address());
+
+    if (peripheral.connect())
+    {
+        Serial.print("Connected: ");
+        Serial.println(peripheral.address());
+    }
+    else
+    {
+        Serial.println("Failed to connect!");
+        return;
+    }
+    
+    peripheral.discoverAttributes();
+    
+    BLECharacteristic ledCharacteristic = peripheral.characteristic("19b10101-e8f2-537e-4f6c-d104768a1214");
+    
+    if (!ledCharacteristic)
+    {
+        //peripheral.disconnect();
+        while(1)
+        {
+        Serial.println("Peripheral does not have LED characteristic!");
+        delay(5000);
+        }
+        return;
+    } 
+      
+      
+      unsigned char ledstate = 0;
+
+    discovered = false;
+    while (peripheral.connected())
+    {
+        if (ledstate == 1)
+        {
+            ledstate = 0;
+        }
+        else
+        {
+            ledstate = 1;
+        }
+        ledCharacteristic.write(&ledstate, sizeof(ledstate));

```

Comment:
Sorry, yes that is correct (I forgot is was the central context). `BLEAttributeWithValue` has a `writeByte` method, this should be used here instead.